libraries/AP_CANManager/AP_CANTester.h

```diff
@@ -18,7 +18,7 @@
 #include "AP_CANDriver.h"
 #include <AP_HAL/Semaphores.h>
 #include <AP_UAVCAN/AP_UAVCAN.h>
-#if HAL_MAX_CAN_PROTOCOL_DRIVERS > 1 && !HAL_MINIMIZE_FEATURES && HAL_ENABLE_LIBUAVCAN_DRIVERS
+#if HAL_MAX_CAN_PROTOCOL_DRIVERS > 1 && !HAL_MINIMIZE_FEATURES && HAL_ENABLE_CANMANAGER
```

Comment:
Our pattern is
`HAL_CANMANAGER_ENABLED`

May as well conform if we're changing it everywhere